bin/fuel-convert

```diff
@@ -24,9 +36,11 @@ if __name__ == "__main__":
             name, parents=[parent_parser],
             help='Convert the {} dataset'.format(name))
         subparser.add_argument(
-            "-o", "--output-file", help="where to save the dataset", type=str,
-            default=os.path.join(os.getcwd(), '{}.hdf5'.format(name)))
+            "-o", "--output", dest="output_directory",
+            help="where to save the dataset", type=str,
```

Comment:
Would something like

    where to save the dataset. Can be either a file or a directory.
    If it's a directory, fuel-convert will save using a dataset-specific
    default file name.

work for you?